CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - `Player.buffer` to control buffer preferences and to query the current buffer state
 - `DownloadFinishedEvent` to signal when the download of specific content has finished
 
+### Fixed
+
+- Android: Playback doesn't pause when app goes to background
```

Comment:
Let's also add a changelog for the destroy function now being called so someone relying on the current behaviour (which I don't suspect but let's rather be safe than sorry) can see that there were changes.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/313/commits/1c2243613d54468c0a6fbedbe2a6b5b7e034bb73